src/components/TextField.tsx

```diff
@@ -52,7 +52,7 @@ function Field(props: TextFieldProps) {
         required={props.required}
         className={classes.textField}
         InputLabelProps={{ shrink: true }}
-        value={props.value || undefined}
```

Comment:
I'm guessing this was for testing and you forgot to revert?? Unless this means we're finally at 100% controlled components 👀

Reply:
Though if it is the case that we've converted all to controlled, we can wrap up #74

Reply:
Not a mistake! We're at 100% controlled components :) (except for 1 TextField in `FinancialInfo` that will get worked on in a couple days). Will close #74 when this is merged!